client/components/upgrades/google-apps/google-apps-dialog/product-details.jsx

```diff
@@ -31,7 +31,7 @@ class GoogleAppsProductDetails extends Component {
 
 					<p>
 						{ translate(
-							"We've partnered with Google to offer you email, storage, docs, calendars " +
+							"We've teamed up with Google to offer you email, storage, docs, calendars, " +
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'We've partnered with Google to offer you email, storage, docs, calendars, and more integrated with your site.' )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/abtest-gsuite-in-signup-flow)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).